### PR TITLE
R novice gapminder/writing data

### PR DESCRIPTION
**Pull request by TomKellyGenetics**
Re-open PR https://github.com/swcarpentry-ja/i18n/pull/50 with conflicts resolved.

### CODE REVIEW

File: po/r-novice-gapminder.ja.po
Comment (TomKellyGenetics) [inline po/r-novice-gapminder.ja.po]:
```suggestion
```

**Comment by TomKellyGenetics** [comment]
@atusy please revert changes that you would have made in review.

File: po/r-novice-gapminder.ja.po
Comment (masamiy) [inline po/r-novice-gapminder.ja.po]: 図の保存
may sound more header-y?

File: po/r-novice-gapminder.ja.po
Comment (masamiy) [inline po/r-novice-gapminder.ja.po:15648]: If you decided to leave as it is, it is fine, but what about this?
"データをクリーンアップするスクリプトを書きましょう。“

Comment (atusy) [reply po/r-novice-gapminder.ja.po:15648]: or what about 整形?

File: po/r-novice-gapminder.ja.po
Comment (atusy) [reply po/r-novice-gapminder.ja.po]: sounds nice

Comment (atusy) [reply po/r-novice-gapminder.ja.po]: This PR is @TomKellyGenetics 's. I guess I have no write access.

File: po/r-novice-gapminder.ja.po
Comment (masamiy) [reply po/r-novice-gapminder.ja.po:15648]: May be we can leave it as it is. I Googled and "キレイ”, ”きれい” or "クリーン” appear to be used interchangeably. It makes sense, which is most important. Thank you for your attention and great work!

File: po/r-novice-gapminder.ja.po
Comment (masamiy) [reply po/r-novice-gapminder.ja.po]: It is not a big-y. We can leave it as it is :)

Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]:
```suggestion
## 図の保存
```

Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po]: Normally I'd do this via the command-line but you can see here using \`\`\`suggestion can propose changes via the GUI that can be accepted.

File: po/r-novice-gapminder.ja.po
Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po:15648]: Had a quick search to see what's used in Tech industry in Japanese.
[Microsoft Office support](https://support.office.com/ja-jp/article/%E3%83%87%E3%83%BC%E3%82%BF%E3%82%92%E3%82%AF%E3%83%AA%E3%83%BC%E3%83%B3%E3%82%A2%E3%83%83%E3%83%97%E3%81%99%E3%82%8B%E6%96%B9%E6%B3%95%E3%83%88%E3%83%83%E3%83%97-10-2844b620-677c-47a7-ac3e-c2e157d1db19) uses クリーンアップ ("clean up"), although I think クリーン is more concise. It seems that [きれい](https://area.autodesk.jp/column/tutorial/maya_atoz/cleaup-edge/) and [綺麗](http://excel.resocia.jp/report/357/) are also used in the context of data formatting. 

As discussed in our [Guidelines](https://github.com/swcarpentry-ja/i18n/blob/ja/TranslatorGuidelines.md) we prefer to use terms so that learners will be able to understand terminology used elsewhere if they look for help. I think the most important thing is to be consistent, so I recommend to decide on one to use throughout the lessons and add it to the [Glossary](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms) on the Wiki pages. Sorry I know this is a lot of information and our documents are a bit out of date.

In this case I think either is fine, although isn't the Kanji form of 綺麗な rarely used (although I can read it so I'm sure a native speaker can). Stylistically, the きれい could be harder to reader with particles?

```suggestion
"データをキレイにするスクリプトを書きましょう。\n"
```

Of course I think Japanese readers are also familiar with クリーニング:

```suggestion
"データをクレーンにするスクリプトを書きましょう。\n"
```

Thanks for your inputs. I'm OK to leave it in Kanji unless anyone thinks either of these is better.

Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po:15648]: Just curious, is there any reason this wasn't used as it's more literal translation of the English text?
```suggestion
"データクリーニングのスクリプトを書きましょう。\n"
```

Comment (TomKellyGenetics) [reply po/r-novice-gapminder.ja.po:15648]: Since both native speakers seem to agree to leave the Kanji form, I'm going to close this.

**Comment by TomKellyGenetics** [comment]
@joelnitta @rikutakei  All changes have been resolved but it seems Mergify hasn't automatically merged as a reviews "suggested changes". As I submitted the PR, I cannot review it myself. Can someone try (Re)Review with "approve" to see if automated merges work?

**Review by joelnitta — APPROVED** [review]
Testing automated merge

**Pull request merged**
